src/dcqc/updaters.py

```diff
@@ -0,0 +1,56 @@
+from csv import DictWriter
+from pathlib import Path
+from typing import Dict, List
+
+from dcqc.parsers import CsvParser
+from dcqc.suites.suite_abc import SuiteABC
+
+
+class CsvUpdater:
+    input_path: Path
+    output_path: Path
+    parser: CsvParser
+
+    def __init__(self, input_path: Path, output_path: Path):
+        self.output_path = output_path
+        self.parser = CsvParser(input_path)
+
+    def update(self, suites: List[SuiteABC]):
+        suite_dict: Dict[
+            str, List[str]
+        ] = {}  # mypy made me do this, but only here. not sure why
+        # {url: [list_of_statuses]} data structure to allow for multi-file targets
+        for suite in suites:
```

Comment:
So with multi file targets, the current logic is that a particular file will have multiple suites.

It doesn't look like you're looping through the multiple files within a suite though (not sure you need to, just an observation)

Reply:
So does that mean that for a file pair, each member of the pair would have an individual suite and then they would both be part of a paired suite? Meaning that the `url` field would have two URLs in that case? Is there an existing example somewhere that I can use?

Reply:
Currently, the functionality isn't fully used, we need to add the ability for DCQC to leverage the multi-file feature that was added.  Lets add a TODO for now.